src/config_files.rs

```diff
@@ -125,3 +150,102 @@ pub(crate) fn create_history_path() -> Option<PathBuf> {
         }
     })
 }
+
+fn default_config_contents() -> &'static str {
+    r#"
+# Nushell Config File
+
+def create_left_prompt [] {
+    let path_segment = ([
+        ($nu.cwd)
+        (char space)
+    ] | str collect)
+
+    $path_segment
+}
+
+def create_right_prompt [] {
+    let time_segment = ([
+        (date now | date format '%m/%d/%Y %I:%M:%S%.3f')
+    ] | str collect)
+
+    $time_segment
+}
+
+# Use nushell functions to define your right and left prompt
+let-env PROMPT_COMMAND = { create_left_prompt }
+let-env PROMPT_COMMAND_RIGHT = { create_right_prompt }
+
+# The prompt indicators are environmental variables that represent
+# the state of the prompt
+let-env PROMPT_INDICATOR = "〉"
+let-env PROMPT_INDICATOR_VI_INSERT = ": "
+let-env PROMPT_INDICATOR_VI_NORMAL = "〉 "
+let-env PROMPT_MULTILINE_INDICATOR = "::: "
+
+let $config = {
+  filesize_metric: $true
+  table_mode: rounded # basic, compact, compact_double, light, thin, with_love, rounded, reinforced, heavy, none, other
+  use_ls_colors: $true
+  rm_always_trash: $false
+  color_config: {
+    separator: yd
+    leading_trailing_space_bg: white
+    header: cb
+    date: pu
+    filesize: ub
+    row_index: yb
+    hints: dark_gray
+    bool: red
+    int: green
+    duration: red
+    range: red
+    float: red
+    string: red
+    nothing: red
+    binary: red
+    cellpath: red
+  }
+  use_grid_icons: $true
+  footer_mode: always #always, never, number_of_rows, auto
+  quick_completions: $false
+  animate_prompt: $false
+  float_precision: 2
+  use_ansi_coloring: $true
+  filesize_format: "b" # b, kb, kib, mb, mib, gb, gib, tb, tib, pb, pib, eb, eib, zb, zib, auto
+  env_conversions: {
+    "PATH": {
+        from_string: { |s| $s | split row (char esep) }
+        to_string: { |v| $v | str collect (char esep) }
+    }
+  }
+  edit_mode: emacs # emacs, vi
+  max_history_size: 10000
+  log_level: error  # warn, error,  info, debug, trace
```

Comment:
I disabled `log_level` in https://github.com/nushell/nushell/pull/4391. I'm not sure it can be re-enabled as a config point but we may be able to enabled a `--log_level blah` functionality as a nu parameter similar to the way 0.44 works.

Reply:
yeah a flag sounds good